.ci/macos/dependencies/gource.sh

```
@@ -0,0 +1,118 @@
+curl -LO https://github.com/acaudwell/Gource/releases/download/gource-0.55/gource-0.55.tar.gz
+tar -xzf gource-0.55.tar.gz
+cd gource-0.55
+./configure
+make
+sudo make install
+
+# link BOOST_ROOT
+curl -LO https://github.com/acaudwell/Gource/releases/download/gource-0.55/gource-0.55.tar.gz && \
+  tar -xzf gource-0.55.tar.gz && \
+  cd gource-0.55 && \
+  ./configure --with-boost=$BOOST_ROOT && \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)